lib/elixir/src/elixir_exp.erl

```diff
@@ -515,13 +515,13 @@ expand_local(Meta, Name, Args, #{module := Module, function := Function} = E) ->
 %% Remote
 
 expand_remote(Receiver, DotMeta, Right, Meta, Args, E, EL) ->
+  {EArgs, EA} = expand_args(Args, E),
   if
     is_atom(Receiver) ->
-      elixir_lexical:record_remote(Receiver, ?m(E, function), ?m(E, lexical_tracker));
+      elixir_lexical:record_remote(Receiver, ?m(E, function), Right, length(EArgs), ?line(Meta), ?m(E, lexical_tracker));
```

Comment:
It seemed right to use `EArgs` here. Presumably if the args contain a call to `unquote_splicing`, the number of args might increase after expansion? Just wanted to confirm.